docs/docs/install/config/kubernetes.md

```diff
@@ -17,22 +17,20 @@ metadata:
 data:
   config.yaml: |-
     cache:
-        type: mem
-        size: 512
+      - cache-type=mem
```

Comment:
These must be the new structured.

Reply:
but that is in your PR, no?

Reply:
Mine (structured config) was already merged. So you're rolling it back hehehe.